src/types/UiaTextRangeBase.cpp

```diff
@@ -9,91 +9,22 @@ using namespace Microsoft::Console::Types;
 using namespace Microsoft::Console::Types::UiaTextRangeBaseTracing;
 
 // toggle these for additional logging in a debug build
+//#define _DEBUG 1

```

Comment:
Doesn't this get defined for you when using a debug build configuration type? Why define it again? To debug in release mode?

Reply:
Unintentionally, yes? I left this here on accident but now, the debug build may be too slow for some UIA Clients. So This may not actually be a bad idea. Keeping for now.